### ISSUE TITLE
Upgrade to Babel6

### ISSUE THREAD

**Issue by nikgraf** [sapegin/react-styleguidist#76]
I couldn't get it up & running due issues with Babel5/6

```
Module build failed: ReferenceError: [BABEL] /my-path/app/node_modules/react-styleguidist/src/index.js: Unknown option: /my-path/.babelrc.presets
```

**Comment by mxstbr** [comment]
:+1:

**Comment by gregz67** [comment]
+1

**Comment by kazooie** [comment]
I forked to use styleguidist with Babel 6, but it is very, very hacky since you now have to compile your own babel to give to the web browser for live previews. I first tried the babel-standalone project (https://github.com/Daniel15/babel-standalone), but couldn't get it working with webpack. 

I made the babel compilation a separate step in bin/styleguidist and pushed a precompiled version that contains es2015, react, and stage-0 presets. 

I don't plan on making this ready for a PR, and only made it as a temporary solution until babel adds some support for babel in the browser again, but maybe it can provide some ideas for how to do handle the upgrade.

https://github.com/kazooie/react-styleguidist/tree/github-feature/babel-6
